src/main/java/com/spotride/spotride/user/dto/UserRequestDto.java

```diff
@@ -0,0 +1,31 @@
+package com.spotride.spotride.user.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * User request DTO model.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserRequestDto {
+
+    private Long id;
+
+    @NotBlank(message = "Username is required")
```

Comment:
NotEmpty. NotBlank allows null values

Reply:
It's not, and additionally it not give put into a field white spaces

![image](https://github.com/user-attachments/assets/051ca3c1-a631-4cae-a4b3-26beaf44a3af)